src/openrct2/actions/TrackSetBrakeSpeedAction.cpp

```diff
@@ -57,7 +57,7 @@ GameActions::Result TrackSetBrakeSpeedAction::QueryExecute(bool isExecuting) con
 
     if (!LocationValid(_loc))
     {
-        return GameActions::Result(GameActions::Status::NotOwned, STR_NONE, STR_NONE);
+        return GameActions::Result(GameActions::Status::NotOwned, STR_CANT_CHANGE_THIS, STR_LAND_NOT_OWNED_BY_PARK);
```

Comment:
While it is better that function doesn't check for the ownership, it checks if the location is generally valid on the map.

Reply:
Good point ZehMatt!!

The `LocationValid` function:

https://github.com/OpenRCT2/OpenRCT2/blob/b5786c2a86489e43bcd0d936b0d19c2336a1a527/src/openrct2/actions/GameAction.cpp#L478-L480

The `MapIsLocationValid` function:

https://github.com/OpenRCT2/OpenRCT2/blob/b5786c2a86489e43bcd0d936b0d19c2336a1a527/src/openrct2/world/Map.cpp#L791-L796

So I think the `Status` should be `InvalidParameters`. I don't see a string for "invalid location" so only using `STR_CANT_CHANGE_THIS` seems ok.

Reply:
Let’s do that, then. You can leave out the last parameter if you don’t set it:

```suggestion
        return GameActions::Result(GameActions::Status::InvalidParameters, STR_CANT_CHANGE_THIS);
```

Reply:
Done! In a different pull request maybe I'll add `STR_ERR_INVALID_LOCATION`. Looks like the build is failing now because of wrong parameter count, I can fix it later

Reply:
It turns out there is already 
```
STR_OFF_EDGE_OF_MAP = 1028
```
which is used in other cases when `LocationValid` fails. I can update this PR later

Reply:
I pushed a commit to add `STR_OFF_EDGE_OF_MAP`, and all the builds succeed now